src/launcher/standalone_launcher/.storybook/preview.js

```diff
@@ -0,0 +1,4 @@
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+}
```

Comment:
Add newline at end of file. (Please set up your IDE to insert these automatically).